### PR TITLE
Mobile Menu (Js Function) - Fourth Milestone

### PR DESCRIPTION
**Pull request by aliatayee**
## Mobile Menu (Js function) - Fourth Milestone
in this milestone we have worked on the following parts:

- Created and structured the js file
- defined the proper function for the hamburger menu
- applied proper styles based on the template in style.css
- created the mobile menu and called the js function
- applied the function on required parts

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @aliatayee, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] As requested by the previous reviewer, Kindly ensure that you set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.
- **Hint** Add `.eslintrc.json` file and make sure that linter is passed in GitHub checks.
- [ ] On the mobile screen, I click on the hamburger menu, the menu displayed, But while switching to the desktop version, it should disappear, Kindly fix this.

rthft


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```
Comment (keroloussamy) [inline .eslintrc.json:25]: - [ ] [OPTIONAL] You have to update your linters.yml with this file [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) too, so GitHub can check it JS too.

**Review by keroloussamy — APPROVED** [review]
Hi @aliatayee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — COMMENTED** [review]
Hi @aliatayee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**